src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <ServerGarbageCollection>false</ServerGarbageCollection>
+    <InvariantGlobalization>true</InvariantGlobalization>
+    <UserSecretsId>b8ffb8d3-b768-460b-ac1f-ef267c954c85</UserSecretsId>
+    <EnableRequestDelegateGenerator>true</EnableRequestDelegateGenerator>
+    <PublishRelease>true</PublishRelease>
```

Comment:
This isn't needed. It is the default for 8.0 projects.

Reply:
It is? I thought it was going to be set in templates rather than defaulted based on TFM, hmmm.

Reply:
https://github.com/dotnet/sdk/pull/29155